### DISCUSSION TITLE
Errors happening: bad file descriptor and eslint not working

### DISCUSSION THREAD

**Discussion by victorreis** [vercel/turborepo#3067] [Help] [answered]
Hi, it's my first time configuring a Turborepo repository. I like using strict sets of eslint rules and other configs to ensure minimum code quality. As I was configuring it two problems started: the message "Parsing error: EBADF: bad file descriptor, lstat '/dev/fd/13' eslint" and the other eslint rules not working. You can see the problem in this repository:

https://github.com/victorreis/stream-dom

Can someone help me?

**Comment by anthonyshew** [comment, marked as answer]
Closing as stale! Hoping you've got this sorted by a year later. 😄

There are many examples of ESLint to take a look at in [the examples directory](https://github.com/vercel/turbo/tree/main/examples)
